applications/asset_tracker_v2/README.rst

```diff
@@ -1,27 +0,0 @@
-.. _asset_tracker_v2:
```

Comment:
I suggest adding a section in the migration guide that explains how to migrate to the add-on solution. This section should use this exact tag, so that any potential links that still remain are redirected to that section.